foreman_remote_execution.gemspec

```diff
@@ -24,7 +24,6 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files = Dir['README*', 'LICENSE']
 
   s.add_dependency 'deface'
-  s.add_dependency 'dynflow', '~> 0.8.26'
   s.add_dependency 'foreman_remote_execution_core'
   s.add_dependency 'foreman-tasks', '~> 0.12'
```

Comment:
I believe you'll need to change this, too.

Reply:
I don't think its strictly necessary is it?

Reply:
Oh right, I hate `~>`

Reply:
If the goal is to mineralize the amount of work, when dynflow raises major version, I'm for using `>= 0.8.26`, just to indicate what is the minimal version, and leaving the `~>` style in foreman only